content/outcomes/application-development/pair-programming.md

```diff
@@ -0,0 +1,128 @@
+- [List at least five potential challenges of pair programming](#list-at-least-five-potential-challenges-of-pair-programming)
+- [Describe the mechanics of the driver/navigator style of pair programming](#describe-the-mechanics-of-the-drivernavigator-style-of-pair-programming)
+
+## Define pair programming and remote pair programming
+Pair programming is a software development technique whereby two programmers work on the same problem, at the same time, on the same computer, with the goal of producing high quality software quickly. 
+
+Regarding “on the same computer,” we do not mean two people hunched over a tiny laptop. When co-located, we recommend setting up two large monitors, two sets of keyboards, two mice, and two sets of anything else the pair needs to work effectively together, such as a laptop docking station. 
+
+If the pair is not co-located, [remote pair programming](https://tanzu.vmware.com/content/blog/5-tips-for-effective-remote-pair-programming-while-working-from-home) is a good alternative. Remote pair programming is pair programming with the help of commonly available collaboration software.
+
+## List at least five potential benefits of pair programming
+“Done” does not mean the developers have committed their code and lifted their fingers off their keyboards. For VMware Tanzu Labs, “done” means low-defect software delivered to users in production that does not need rework due to errors or missed requirements. Pair programming helps accomplish this mission in the following ways:
+
+### Code quality benefits
+The most beneficial outcome of pair programming is the quality of the code written by the team. As pairs switch (often daily), everyone has a chance to contribute  and refine the system they are developing. Benefits include:
+
+*   Fewer bugs
+*   Better system design
+*   More understandable code
+*   Wider code testing coverage
+*   “Fresh eyes” to help expose hidden flaws
+
+### Time-to-production benefits
+While pair programming does support shipping high quality code to production quickly, predictability is a more important benefit over speed. Radical swings in delivery velocity introduce risk into the development cycles — pair programming helps mitigate swings in delivery velocity in the following ways: 
+
+*   Intense focus on the task at hand
+*   Less rework
+*   More accurate implementation of requirements
+*   Less time in QA
+
+### Knowledge sharing benefits
+While pair programming, not only are two people working on the same difficult problem together, they are also constantly teaching one another in the following ways: 
+
+*   New team member onboarding
+*   Junior developers learning from senior developers
+*   Senior developers learning from junior developers — it’s true!
+*   Breaking down knowledge silos
+*   Reduce the [“bus factor” or “lottery factor”](https://en.wikipedia.org/wiki/Bus_factor)
+*   Tips and tricks sharing
+
```

Comment:
Maybe we can mention the "no superheroes" and evolution to evolve towards a learning organization.